Fixes #4939 - Add Leeds East (EGCM) SMR

rishab-alt opened the pull request:
Fixes #4939 

# Summary of changes
as per issue

# Screenshots (if necessary)
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/60842647/0276652e-f0ad-4d56-bee5-be0f83692efd)

the colors i need opinions on as i have doubts on the runway disused color and the taxiway colors

PLM1995 commented:
Personally not a big fan of the bright red, I don't see any reasonall disused areas shouldn't be the same colour? WHat colour that is I'm not sure.

One option is also to add 'X' labels to denote no-entry bits.

I would have the taxiway between A3 and B3 'over' the disused bit too so it's obvious it can be used as a taxiway.

And changelog please :)

rishab-alt commented:
right... what i have done is that i have changed the disused runway color to what the disused taxiways are and added the crossing that goes over the disused runway @PLM1995 
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/60842647/7b7128af-86cd-4e02-a347-087367374ff5)

PLM1995 commented on Airports/EGCM/SMR/Geo.txt:
```diff
@@ -0,0 +1,76 @@
+N053.50.08.288 W001.11.57.698 N053.50.07.711 W001.11.55.946 smrWhite
+N053.50.07.711 W001.11.55.946 N053.50.07.738 W001.11.55.830 smrWhite
+
+;Holds
+; - F1
+N053.50.01.607 W001.11.59.518 N053.50.01.884 W001.11.58.804 smrWhite
+; - A1
+N053.49.52.516 W001.12.24.144 N053.49.52.479 W001.12.23.315 smrWhite
+; - A2
+N053.50.09.551 W001.11.54.493 N053.50.09.401 W001.11.53.695 smrWhite
+; - A3
+N053.50.12.116 W001.11.53.207 N053.50.11.817 W001.11.52.481 smrWhite
+; - B3
+N053.50.14.794 W001.11.46.302 N053.50.14.291 W001.11.46.015 smrWhite
+; - B2
+N053.50.15.670 W001.11.42.198 N053.50.15.258 W001.11.41.914 smrWhite
+; - B1
+N053.50.23.656 W001.11.05.825 N053.50.23.954 W001.11.05.057 smrWhite
+; - E2
+N053.50.13.815 W001.11.42.353 N053.50.14.257 W001.11.41.958 smrWhite
+; - E1
+N053.50.11.188 W001.11.37.752 N053.50.11.454 W001.11.37.166 smrWhite
```
Single lines - comments at end of line (the ` - ` is unnecessary), and ensure alphabetical order.

PLM1995 commented on Airports/EGCM/SMR/Labels.txt:
```diff
@@ -0,0 +1,10 @@
+"A1" N053.49.52.489 W001.12.23.047 standHold
+"A2" N053.50.09.368 W001.11.53.407 standHold
+"A3" N053.50.11.685 W001.11.52.192 standHold
+"B3" N053.50.14.116 W001.11.46.234 standHold
+"E2" N053.50.13.566 W001.11.42.630 standHold
+"B2" N053.50.15.000 W001.11.41.874 standHold
+"B1" N053.50.23.321 W001.11.06.308 standHold
+"F1" N053.50.01.949 W001.11.58.454 standHold
+"E1" N053.50.10.796 W001.11.38.574 standHold
```
Alphabetical order

PLM1995 commented on .github/CHANGELOG.md:
```diff
@@ -3,6 +3,7 @@
 2. Enhancement - Added Derby (EGBD) SMR - thanks to @rishab-alt
 3. Enhancement - Removed Manchester (EGCC) disused stands - thanks to @rishab-alt
 4. Procedure Change (2309) - Amended EuroCenter vACC EURM-W and EURW-N callsigns to EUC-MW and EUC-WN, respectively - thanks to @hsugden (Harry Sugden)
+5. Enhancement - Add Leeds East (EGCM) SMR - thanks to @rishab-alt
```
```suggestion
5. Enhancement - Added Leeds East (EGCM) SMR - thanks to @rishab-alt
```
Tense

PLM1995 commented on Airports/EGCM/SMR/Geo.txt:
```diff
@@ -0,0 +1,67 @@
+N053.50.12.116 W001.11.53.207 N053.50.11.817 W001.11.52.481 smrWhite ;A3
+N053.50.23.656 W001.11.05.825 N053.50.23.954 W001.11.05.057 smrWhite ;B1
+N053.50.15.670 W001.11.42.198 N053.50.15.258 W001.11.41.914 smrWhite ;B2
+N053.50.14.794 W001.11.46.302 N053.50.14.291 W001.11.46.015 smrWhite ;b3
```
```suggestion
N053.50.14.794 W001.11.46.302 N053.50.14.291 W001.11.46.015 smrWhite ;B3
```
